packages/react/test/specs/components/Tooltip/Tooltip-test.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+
+import Tooltip from 'src/components/Tooltip/Tooltip'
+
+import { mountWithProvider } from '../../../utils'
+
+describe('Tooltip', () => {
+  describe('onOpenChange', () => {
+    test('is called on hover', () => {
+      const spy = jest.fn()
+
+      mountWithProvider(<Tooltip trigger={<button />} content="Hi" onOpenChange={spy} />)
+        .find('button')
+        .simulate('mouseEnter')
+
+      expect(spy).toHaveBeenCalledTimes(1)
+      expect(spy.mock.calls[0][1]).toMatchObject({ open: true })
```

Comment:
```js
expect(onOpenChange).toHaveBeenCalledWith(
  expect.objectContaining({ type: 'mouseEnter' }),
  expect.objectContaining({ open: true }),
)
```

There is a dedicated assertion

Reply:
Unfortunately, with these changes the tests are time-outing, not sure why.. Let's investigate this tomorrow.

Reply:
```js
expect(onOpenChange).toHaveBeenCalledTimes(1)
expect(onOpenChange).toHaveBeenCalledWith(
  expect.objectContaining({ type: 'mouseenter' }),
  expect.objectContaining({ open: true }),
)
```

There was typo in `type`. Tests timeouts because Jest tries to pretty print giant object with an event